src/common/CudaDataUtils.hpp

```diff
@@ -42,6 +42,37 @@ __device__ __forceinline__ unsigned long long device_timer()
   return global_timer;
 }
 
+/*!
+ * \brief Method for launching a CUDA kernel with given configuration.
+ *
+ *        Note: method checks whether number of args and their types in 
+ *              kernel signature matches args passed to this method.
+ */
+template <typename... Args, typename...KernArgs>
+void RPlaunchCudaKernel(void (*kernel)(KernArgs...),
+                        const dim3& numBlocks, const dim3& dimBlocks,
+                        std::uint32_t sharedMemBytes, cudaStream_t stream,
+                        Args const&... args)
+{
+  static_assert(sizeof...(KernArgs) == sizeof...(Args),
+                "Number of kernel args doesn't match what's passed to method");
+
+  using int_array = int[];
+  int_array ia = {[](){
```

Comment:
Using the alias let us avoid naming the array, but using `RAJA_UNUSED_VAR(ia);` is a bit more clear. Remove redundant alias.
```suggestion
  int ia[] = {[](){
```

Reply:
@rhornung67 Everything should be good to go after this.